src/SendHeaders.php

```diff
@@ -30,7 +30,7 @@ public function getContext(array $headers): array
 
 	public function setCacheHeader(): void
 	{
-		if (wp_get_environment_type() != 'production') {
+		if (wp_get_environment_type() === 'local' || wp_get_environment_type() === 'development') {
```

Comment:
Do we definitely not want this available on staging?

Reply:
It's because this was deployed in a staging env that we saw the headers being set on preview pages. So, this way is safer in terms of avoiding any future deploys of potential security related bugs, but the original check was more flexible...